src/trace.ml

```diff
@@ -8,8 +8,10 @@ let supports_command command =
   Lazy.from_fun (fun () ->
       match Core_unix.fork () with
       | `In_the_child ->
-        Core_unix.close Core_unix.stdout;
-        Core_unix.close Core_unix.stderr;
+        (* gracefully hide perf outputs *)
```

Comment:
We should leave a comment here about what versions of perf we observed breakages when close-ing and under what circumstances, because it was surprising.